### PR TITLE
Add daemonset for using --version=latest in COS GPU driver installers

### PR DESCRIPTION
**Pull request by arnav-kansal**

### CODE REVIEW

**Comment by arnav-kansal** [comment]
Hi @pradvenkat, @rkolchmeyer!

These changes add a daemonset to enable the use of --version=latest functionality offered by COS gpu installers.
PTAL.

Thanks,
Arnav

File: nvidia-driver-installer/cos/daemonset-preloaded-latest.yaml [hunk 1]
```diff
@@ -0,0 +1,107 @@
+# Copyright 2017 Google Inc. All rights reserved.
```
Comment (rkolchmeyer) [inline nvidia-driver-installer/cos/daemonset-preloaded-latest.yaml:1]: Update copyright to 2022?

File: nvidia-driver-installer/cos/daemonset-preloaded-latest.yaml [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+          - name: VULKAN_ICD_DIR_HOST
+            value: /home/kubernetes/bin/nvidia/vulkan/icd.d
+          - name: VULKAN_ICD_DIR_CONTAINER
+            value: /etc/vulkan/icd.d
+          - name: ROOT_MOUNT_DIR
+            value: /root
+          - name: COS_TOOLS_DIR_HOST
+            value: /var/lib/cos-tools
+          - name: COS_TOOLS_DIR_CONTAINER
+            value: /build/cos-tools
```
Comment (rkolchmeyer) [inline nvidia-driver-installer/cos/daemonset-preloaded-latest.yaml:92]: nit; other lists don't put 2 spaces before each entry (see volumeMounts). Maybe pick one indentation format?

**Review by rkolchmeyer — APPROVED** [review]
/lgtm

Did you run this on a cluster and verify that it works?

**Comment by arnav-kansal** [comment]
> /lgtm
> 
> Did you run this on a cluster and verify that it works?

Tested on a cluster.

logs from the container:

```
00:04.0 3D controller: NVIDIA Corporation Device 1eb8 (rev a1)
I0302 18:13:47.941790  312987 installer.go:401] Getting the latest GPU driver version
I0302 18:13:47.942196  312987 utils.go:88] Downloading gpu_latest_version from https://storage.googleapis.com/cos-tools/16108.534.18/gpu_latest_version
..
..
```

**Review by pradvenkat — APPROVED** [review]

**Pull request merged**